README.rst

```diff
@@ -28,39 +19,61 @@ locations.
 We are continuously working on new features, such as detecting copyright changes
 and detecting package version changes.
 
+Build and tests status
+======================
 
-Quick Start
-===========
-For more comprehensive installation instructions and development instructions, see:
-`Comprehensive Installation <https://github.com/nexB/deltacode/wiki/Comprehensive-Installation>`_
+We run tests on each commit on multiple CIs to ensure a good platform
+compatibility with multiple versions of Windows, Linux and macOS.
 
-For development instructions, see:
-`Development Instructions <https://github.com/nexB/deltacode/wiki/Development>`_
++--------------+--------------+
+| **Azure**    | **RTD Build**|
++==============+==============+
+|    |azure|   | |docs-rtd|   |
++--------------+--------------+
 
-Make sure you have Python 3.8+ installed:
-  * Download and install Python 3.8+ Windows
-    https://www.python.org/downloads/windows/
-  * Download and install Python 3.8+ for Mac
-    https://www.python.org/downloads/mac-osx/
-  * Download and install Python 3.8+ for Linux via distro package manager
+Documentation
+=============
 
-Next, download and extract the latest DeltaCode release from::
+The DeltaCode documentation is hosted at
+`deltacode.readthedocs.io <https://deltacode.readthedocs.io/en/latest/>`_.
 
-    https://github.com/nexB/deltacode/releases/
+Installation
+============
 
-Open a terminal, extract the downloaded release archive, then `cd` to
-the extracted directory and run this command to display the command
-help. DeltaCode will self-configure if needed::
+Before installing DeltaCode make sure that you have installed the prerequisites
+properly. This means installing Python 3.8 for x86/64 architectures.
+We support Python 3.8, 3.9 and 3.10.
 
-    ./deltacode --help
+See `prerequisites <https://deltacode.readthedocs.io/en/latest/comprehensive_installation.html#prerequisites>`_
+for detailed information on the support platforms and Python versions.
 
-Run a sample delta
+There are a few common ways to `install DeltaCode <https://deltacode.readthedocs.io/en/latest/comprehensive_installation.html>`_.
 
-    ./deltacode -n samples/samples.json -o samples/samples.json
+- `Development installation from source code using a git clone
+  <https://deltacode.readthedocs.io/en/latest/comprehensive_installation.html#source-code-install>`_
+
+- `Development installation as a library with "pip install deltacode"
+  <https://deltacode.readthedocs.io/en/latest/comprehensive_installation.html#pip-install>`_
+
+- `Run in a Docker container with a git clone and "docker run"
+  <https://deltacode.readthedocs.io/en/latest/comprehensive_installation.html#docker-install>`_
+
+
+Quick Start
```

Comment:
I think there should be a sentence mentioning to run `./configure` or `./configure --dev` prior to running `deltacode`

Reply:
The above section of `Quick Start` is `Installation`. It has all the installation guides and I have linked it to the documentation page. See: `Development installation from source code using a git clone`.

I have kept the structure same as scancode-toolkit. Let me know what do you think?

Reply:
Ah, I see what you mean. That makes sense.